Build: build msg about missing template type

Building with Qt 5.13 the compiler message appears:
..\..\..\..\Seamly2D\src\libs\vwidgets\vmaingraphicsview.h:206:10: error: 'unique_ptr' in namespace 'std' does not name a template type

--- a/src/libs/vwidgets/vmaingraphicsview.h
+++ b/src/libs/vwidgets/vmaingraphicsview.h
@@ -64,6 +64,7 @@
 #include <QRubberBand>
 #include <QColor>
 #include <Qt>
+#include <QSharedPointer>
 
 /*!
  * This class adds ability to zoom QGraphicsView using mouse wheel. The point under cursor
@@ -203,7 +204,7 @@ protected:
     virtual void          mouseReleaseEvent(QMouseEvent *event) Q_DECL_OVERRIDE;
     virtual void          mouseDoubleClickEvent(QMouseEvent *event) Q_DECL_OVERRIDE;
 
-    std::unique_ptr<QCursor> curMagnifier;
+    QSharedPointer<QCursor> curMagnifier;
 
 private:
     Q_DISABLE_COPY(VMainGraphicsView)

